src/braket/ir/jaqcd/shared_models.py

```diff
@@ -152,6 +167,35 @@ class TwoDimensionalMatrix(BaseModel):
     )
 
 
+class TwoDimensionalMatrixList(BaseModel):
+    """
+    List of Two dimensional non-empty matrices.
```

Comment:
Two shouldn't be capitalized.

Also, it should be "two-dimensional". (Please fix below in comments for matrix as well.)